04 Strategy Library/Fama French Five Factors/01 Introduction.html

```diff
@@ -0,0 +1,3 @@
+<p>
+    At QuantConnect, we seek to make financial models ever more accessible to our community. This article will walk through the implementation of a stock selection strategy based on the popular Fama French five-factor financial model.
```

Comment:
Please remove the sales language. These tutorials are intended to be scientific method demos. Simply state what the five factor financial model is and when it was created. You have pieces of this in the next section